includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -237,46 +262,62 @@ private function getLoggedInUserData( $user ) {
 			'tracking_label' => 'account.sign-out',
 		];
 
+		if ( $this->hasAuthorProfile( $user ) && !empty( $wgEnableAuthorProfileLinks ) ) {
```

Comment:
Can we change the order in this condition? Checking global variable value is more lightweight operation.